content/en/docs/setup/independent/high-availability.md

```diff
@@ -42,7 +42,7 @@ For each master that has been provisioned, follow the [installation guide](/docs
 
 For highly available setups, you will need to decide how to host your etcd cluster. A cluster is composed of at least 3 members. We recommend one of the following models:
 
-1. Hosting etcd cluster on separate compute nodes (Virtual Machines), or
+1. Hosting etcd cluster on separate compute nodes (Virtual Machines)
```

Comment:
Good catch -- but would also be better as an unordered (bulleted) list, since it's not a set of steps you perform in order but two alternatives.

Reply:
ok, i got this and i'm curious about your timezone

Reply:
@Bradamant3 fixed now